customtkinter/windows/widgets/ctk_combobox.py

```diff
@@ -18,32 +23,32 @@ class CTkComboBox(CTkBaseClass):
     """
 
     def __init__(self,
-                 master: any,
+                 master: CTkBaseClass,
                  width: int = 140,
                  height: int = 28,
-                 corner_radius: Optional[int] = None,
-                 border_width: Optional[int] = None,
-
-                 bg_color: Union[str, Tuple[str, str]] = "transparent",
-                 fg_color: Optional[Union[str, Tuple[str, str]]] = None,
-                 border_color: Optional[Union[str, Tuple[str, str]]] = None,
-                 button_color: Optional[Union[str, Tuple[str, str]]] = None,
-                 button_hover_color: Optional[Union[str, Tuple[str, str]]] = None,
-                 dropdown_fg_color: Optional[Union[str, Tuple[str, str]]] = None,
-                 dropdown_hover_color: Optional[Union[str, Tuple[str, str]]] = None,
-                 dropdown_text_color: Optional[Union[str, Tuple[str, str]]] = None,
-                 text_color: Optional[Union[str, Tuple[str, str]]] = None,
-                 text_color_disabled: Optional[Union[str, Tuple[str, str]]] = None,
-
-                 font: Optional[Union[tuple, CTkFont]] = None,
-                 dropdown_font: Optional[Union[tuple, CTkFont]] = None,
-                 values: Optional[List[str]] = None,
+                 corner_radius: int | None = None,
+                 border_width: int | None = None,
+
+                 bg_color: str | tuple[str, str] = "transparent",
+                 fg_color: str | tuple[str, str] | None = None,
+                 border_color: str | tuple[str, str] | None = None,
+                 button_color: str | tuple[str, str] | None = None,
+                 button_hover_color: str | tuple[str, str] | None = None,
+                 dropdown_fg_color: str | tuple[str, str] | None = None,
+                 dropdown_hover_color: str | tuple[str, str] | None = None,
+                 dropdown_text_color: str | tuple[str, str] | None = None,
+                 text_color: str | tuple[str, str] | None = None,
+                 text_color_disabled: str | tuple[str, str] | None = None,
+
+                 font: tuple[Any, ...] | CTkFont | None = None,
+                 dropdown_font: tuple[Any, ...] | CTkFont | None = None,
+                 values: list[str] | None = None,
                  state: str = tkinter.NORMAL,
```

Comment:
Since you're fixing typing: to match Tkinter, this should be `state: Literal["normal", "disabled", "readonly"]`.  Also `justify` should be `justify: Literal["left", "center", "right"]`.